projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -152,6 +152,44 @@ public function is_task_list_enabled( $id ) {
 		return $this->load_value_from_callback( $task_list, 'is_enabled_callback', null );
 	}
 
+	/**
+	 * Check if a task list was dismissed by the user.
+	 *
+	 * @param string $id Task List id.
+	 * @return bool|null True if dismissed, false if not.
+	 */
+	public function is_task_list_dismissed( $id ) {
+		$task_list_dismissed_status = $this->get_task_list_dismissed_status();
+
+		// Return true if the task list is on the dismissed status array and its value is true.
+		return isset( $task_list_dismissed_status[ $id ] ) && $task_list_dismissed_status[ $id ];
```

Comment:
Do we want an explicit `true` check for the value?
```suggestion
		return isset( $task_list_dismissed_status[ $id ] ) && true === $task_list_dismissed_status[ $id ];
```

Reply:
I've added this check just in case someone fiddles with this option value from outside the tasklist api.